src/appshell/internal/startupscenario.cpp

```diff
@@ -20,35 +20,29 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-#include "startupmodel.h"
+#include "startupscenario.h"
 
 using namespace mu::appshell;
 using namespace mu::actions;
 
 static const std::string HOME_URI("musescore://home");
 static const std::string NOTATION_URI("musescore://notation");
 
-StartupModel::StartupModel(QObject* parent)
-    : QObject(parent)
+void StartupScenario::setStartupScorePaths(const io::paths& paths)
 {
+    m_startupScorePaths = paths;
 }
 
-std::string StartupModel::startupPageUri() const
+void StartupScenario::run()
 {
-    switch (configuration()->startupSessionType()) {
-    case StartupSessionType::StartEmpty:
-    case StartupSessionType::StartWithNewScore:
-        return HOME_URI;
-    case StartupSessionType::ContinueLastSession:
-    case StartupSessionType::StartWithScore:
-        return NOTATION_URI;
-    }
+    if (!m_startupScorePaths.empty()) {
+        for (const io::path& path : m_startupScorePaths) {
+            openScore(path);
```

Comment:
We can only open one score

Reply:
@igorkorsukov fixed here:
https://github.com/musescore/MuseScore/pull/8390/commits/075012e75db8d44ca62ab858af44ec6b6367477b